apps/admin-gui/src/app/shared/components/set-login-dialog/set-login-dialog.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="vo-theme">
+  <h1 class="mat-dialog-title">{{'DIALOGS.SET_LOGIN.TITLE' | translate}}</h1>
+  <div *ngIf="!processing" class="dialog-container" mat-dialog-content>
+    <app-perun-web-apps-login-for-namespace
+      [formGroup]="formGroup"
+      (namespaceChanged)="selectedNamespace = $event">
+    </app-perun-web-apps-login-for-namespace>
+  </div>
+  <mat-spinner *ngIf="processing" class="ml-auto mr-auto"></mat-spinner>
+  <div *ngIf="!processing" mat-dialog-actions>
```

Comment:
I believe we don't hide the dialog actions, just disable the "action" button.

Reply:
Fixed.
(NOTE: I used the `create-service-member-dialog` as a template, so we sometimes use it, but I agree that just disabling it is a better solution.)